core/src/main/resources/lib/layout/tasks.jelly

```diff
@@ -34,7 +34,7 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define">
-	<div id="tasks">
+	<div id="tasks" role="navigation">
```

Comment:
If we prefer semantic tags over roles as suggested in [Félix's comment](https://github.com/jenkinsci/jenkins/pull/4914/files#r477363651), this should be `<nav id="tasks">`.